src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,77 @@
+import classNames from 'classnames';
+import React, { useEffect } from 'react';
+import { TodoStatus } from '../../types/TodoStatus';
+
+type Props = {
+  onStatusChange: (status: TodoStatus) => void;
+  activeTodos: number;
+  status: TodoStatus;
+  completedTodos: number;
+  onClear: (clear: boolean) => void;
```

Comment:
Same here, `onClear` sounds like it should set `false` right away so you don't have to specify this false
```suggestion
  onClear: () => void;
```

Reply:
It's for my state variable, i need this to change state. Or i'm wrong?